File: robolectric-tests/src/test/java/com/example/activity/DeckardActivityTest.java
package com.example.activity;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.Robolectric;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.annotation.Config;

import static org.junit.Assert.assertTrue;

@RunWith(RobolectricTestRunner.class)
@Config(manifest = "../application/src/main/AndroidManifest.xml", emulateSdk = 18)
public class DeckardActivityTest {

    @Test
    public void testSomething() throws Exception {
        assertTrue(Robolectric.buildActivity(DeckardActivity.class).create().get() != null);
    }
}

File: robolectric-tests/src/test/kotlin/DeckardActivityTest.kt
package com.example.activity

import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.Robolectric
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

import org.junit.Assert.assertTrue
import kotlin.test.assertFalse

RunWith(javaClass<RobolectricTestRunner>())
Config(manifest = "../application/src/main/AndroidManifest.xml", emulateSdk = 18)
public class DeckardActivityTest {

    Test
    throws(javaClass<Exception>())
    public fun testSomething() {
        assertTrue(Robolectric.buildActivity<DeckardActivity>(javaClass<DeckardActivity>()).create().get() != null)
    }
}
